packages/schematics/angular/workspace/schema.json

```diff
@@ -36,6 +36,12 @@
       "description": "Creates a workspace with stricter TypeScript compiler options.",
       "type": "boolean",
       "default": false
+    },
+    "packageManager": {
+      "description": "The package manager used to install dependencies.",
+      "type": "string",
+      "enum": ["npm", "yarn", "pnpm", "cnpm"],
+      "visible": false
```

Comment:
Can you please remove `"visible"` option? We'd like this to be visible.

Reply:
Done